diffhtwo/experimental/tests/test_bimodal_sfh_opt.py

```diff
@@ -58,8 +58,8 @@ def test_bimodal_sfh_opt():
         t_obs,
         k_SF,
         k_Q,
-        n_steps=1000,
+        n_steps=10,
         step_size=1e-8,
     )
-
-    assert np.allclose(theta_true, theta_fit, atol=1e-1)
+    assert np.float64(losses[-1]) < np.float64(losses[0])
```

Comment:
Likewise, this change also makes sense for purposes of speeding up the test while retaining its core benefits.